.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
Could you maybe add this to your own global gitignore instead of adding it to all projects?

Reply:
TIL there is a global gitignore!! updated